2003/kangraemin/app/src/main/java/com/example/kangraemin/adapter/SearchResultAdapter.kt

```diff
@@ -31,6 +31,10 @@ class SearchResultAdapter :
         return data.size
     }
 
+    fun setData(data: ArrayList<MovieDetail>) {
+        this.data = data
```

Comment:
```suggestion
        this.data.clear()
        this.data.addAll(data)
```
이렇게 해주면 매번 새로운 리스트를 할당하지 않을 수 있어요.

Reply:
수정하였습니다.
1dc0229